test/stats/test_stats.py

```diff
@@ -192,3 +188,38 @@ def test_decimal_format():
 
     assert str(Stats(1.0, 0.12, 0.5)) == "1.00 ± 0.12 [σ²=0.50]"
     assert str(Stats(1.0, 0.12, 0.5, R_hat=1.01)) == "1.00 ± 0.12 [σ²=0.50, R̂=1.0100]"
+
+
+@flaky
```

Comment:
uhm are we sure this is a good idea?

Reply:
if we start systematically re-running tests we will maybe spend an (even more) considerable amount of time in the test suits and have highly non-deterministic things around .... :)

Reply:
I've run this test quite a few times locally and it did not fail for the chains with 1000 of Gaussian samples. So this should occur only very rarely at all and if it happens, `flaky` will only re-try it once, which is pretty fast.

I think its an improvement compared to us sometimes re-running the whole test suite when one of the non-deterministic VMC tests fails (which is also rare, but happens). I think in those case we would actually benefit from adding the `flaky` decorator there.